data/stoa0171/stoa001/stoa0171.stoa001.opp-lat3.xml

```diff
@@ -46,28 +48,28 @@
               <editor>
                 <persName>
                   <name xml:lang="lat">Samuel Brandt</name>
-                  <addName xml:lang="de">Samuel Brandt</addName>
+                  <addName xml:lang="deu">Samuel Brandt</addName>
                 </persName>
               </editor> 
               <editor>
                 <persName>
                   <name xml:lang="lat">Georgius Laubmann</name>
-                  <addName xml:lang="de">Georg Laubmann</addName>
+                  <addName xml:lang="deu">Georg Laubmann</addName>
```

Comment:
Here I would just keep Georg Laubmann (or G. Laubmann in WorldCat) with no language attribute.